blog/langlydz-part-ilewen/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Langłydż - part ilewen: żargon, zapożyczenia i polglisz'
+authors: pensjonatus
+date: '2024-12-21'
+tags:
+  - 'warsztat'
+  - 'z-przymrużeniem-oka'
+  - 'humor'
+  - 'język'
+coverImage: 'polglisz.png'
+---
+
+Ludzie czasem hejtują na korpomowę. Ja tego nie robię i to serio, bez ironii.
+Korpomowa to nie jest jakaś orwellowska nowomowa. Korpomowa to coś bardzo
+przydatnego: **żargon**.
+
+<!--truncate-->
+
+## Po co jest żargon
+
+![Dwaj przyjaciele na wakacjach, jeden mówi "co mamy dzisiaj na agendzie?" a drugi odpowiada śmiejąc się: "hahaha, nie wiem"](./images/agenda.png)
+
+Żargon, w tym żargon oparty o zapożyczenia, jest jak najbardziej w porządku.
+Ułatwia i przyspiesza komunikację, a dodatkowo służy do identyfikacji zawodowej
+(mówimy i czujemy się jak członkowie grupy). Żargon jest ładny, tak jak żółty
+kask jest ładny: bo pasuje do okazji. Na budowie używasz żółtego kasku, a w
+pracy używasz żargonu.
+
+Przykład: Robimy PRy do repo żeby rozwiązać issue. Jak nafarmimy quick winów, to
+będzie o czym opowiadać na meetupie. Dobrze się składa, bo otworzyli call for
+papers.
+
+Jak jest _call for papers_ po polsku? Nie wiem, bo nie brałem udziału w
+konferencji akademickiej od czasów studiów. Szybciej jest powiedzieć call for
+papers, niż tworzyć sztuczne zamienniki, albo robić yyy i aaa próbując sobie
+przypomnieć jakieś archaiczne polskie słowo.
+
+## Przepraszam, czy mogę pożyczyć słówko?
+
+Zapożyczenia, nie tylko żargonowe, ale również te w mowie powszechnej, ogólnie
+wzbogacają język. Bez nich byśmy nie mieli fryzjera, dachu, czy meczu.
+
+Nawet zapożyczone znaczenia są fajne; kontrybucja nie musi się kojarzyć tylko z
+wojną, ale może oznaczać pozytywny wkład. Pasjonat nie musi wpadać w złość, może
+po prostu bardzo coś lubić. A kolaboracja nie musi oznaczać przejścia na stronę
+wroga.
+
+Jestem po studiach językoznawczych i mam silne przekonanie, że mój punkt
```

Comment:
Ja bym osobiście nie napisał czegoś takiego, bo trochę tworzysz sytuację "ja tu jestem ekspertem, więc nie mamy o czym gadać" :) Może trzeba to trochę złagodzić, żeby czytelnik nie poczuł się traktowany z góry, bo ma inne zdanie niż Ty.

```suggestion
Mam silne przekonanie, że mój punkt
```